enclave/core/sgx/calls.c

```diff
@@ -484,6 +484,17 @@ OE_INLINE void _handle_oret(
     td->oret_result = result;
     td->oret_arg = arg;
 
+    /* Restore the FXSTATE and flags */
+    asm volatile("pushq %[rflags] \n\t" // Restore flags.
+                 "popfq \n\t"
+                 "fldcw %[fcw] \n\t"     // Restore x87 control word
+                 "ldmxcsr %[mxcsr] \n\t" // Restore MXCSR
+                 : [mxcsr] "=m"(callsite->mxcsr),
+                   [fcw] "=m"(callsite->fcw),
+                   [rflags] "=m"(callsite->rflags)
+                 :
+                 :);
```

Comment:
Since flags register is being changed in the assembly snippet, "flags" or "cc" ought to be added to the clobber list for correctness though it may not affect the generated code in this case.

Reply:
It doesn't seem to generate different assembly, but I added "cc" to the clobber list.